src/partition/tests/ReceivedPartitionTest.cpp

```diff
@@ -1066,6 +1069,414 @@ BOOST_AUTO_TEST_CASE(TestCompareBoundingBoxes3D)
   tearDownParallelEnvironment();
 }
 
+template <typename T>
+void testParallelSetOwnerInformation(T &mesh, int dimensions)
+{
+  bool   flipNormals  = true;
+  double safetyFactor = 0;
+
+  testing::ConnectionOptions options;
+  options.useOnlyMasterCom = false;
+  options.useTwoLevelInit  = true;
+  options.type             = testing::ConnectionType::PointToPoint;
+
+  auto                                      participantCom = com::PtrCommunication(new com::SocketCommunication());
+  m2n::DistributedComFactory::SharedPointer distrFactory;
+
+  auto m2n = m2n::PtrM2N(new m2n::M2N(participantCom, distrFactory, options.useOnlyMasterCom, options.useTwoLevelInit));
+
+  mapping::PtrMapping boundingFromMapping = mapping::PtrMapping(new mapping::NearestNeighborMapping(mapping::Mapping::CONSISTENT, dimensions));
+  mapping::PtrMapping boundingToMapping   = mapping::PtrMapping(new mapping::NearestNeighborMapping(mapping::Mapping::CONSERVATIVE, dimensions));
+  boundingFromMapping->setMeshes(mesh, mesh);
+  boundingToMapping->setMeshes(mesh, mesh);
+
+  ReceivedPartition part(mesh, ReceivedPartition::ON_SLAVES, safetyFactor);
+  part.addM2N(m2n);
+
+  part.addFromMapping(boundingFromMapping);
+  part.addToMapping(boundingToMapping);
+
+  part._mesh->computeBoundingBox();
```

Comment:
Can't you call this directly on the mesh instead going through the partition?

Reply:
yes, I can
done!